components/scream/src/share/io/scream_scorpio_interface.hpp

```diff
@@ -23,6 +23,8 @@ static constexpr int PIO_INT = 4;
 namespace scream {
 namespace scorpio {
 
+  using offset_t = std::int64_t;
```

Comment:
Neither here nor there, but `size_t` and `ssize_t` (its signed equivalent) kind of serve the same purpose, if you had any reservations about adding a new type alias.

Reply:
In F90 they use unsigned integers. If I want to pass a ptr to our offsets arrays to scorpio, I think I need unsigned as well.